src/shared/lib/PharmacoDBUtils.ts

```diff
@@ -0,0 +1,65 @@
+import * as _ from 'lodash';
+import {
+    IPharmacoDBGeneDrugAssociationData,
+    IPharmacoDBView,
+    IPharmacoDBViewList,
+    IPharmacoDBCnaRequest,
+} from 'shared/model/PharmacoDB.ts';
+import pharmacoDBClient from 'shared/api/PharmacoDBClientInstance';
+
+/**
+ * Asynchronously return a map with PharmacoDB information from the oncoTreeCode, genes and CNA Viewdata given.
```

Comment:
I can't figure out your oncoTreeCode question either. I'll update this comment when I (or someone I asked) figure it out.